x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -344,57 +358,81 @@ export const visualizationTypes: VisualizationType[] = [
   {
     id: 'bar',
     icon: 'visBarVertical',
-    largeIcon: chartBarSVG,
+    largeIcon: LensIconChartBar,
     label: i18n.translate('xpack.lens.xyVisualization.barLabel', {
       defaultMessage: 'Bar',
     }),
   },
   {
     id: 'bar_horizontal',
     icon: 'visBarHorizontal',
-    largeIcon: chartBarHorizontalSVG,
+    largeIcon: LensIconChartBarHorizontal,
     label: i18n.translate('xpack.lens.xyVisualization.barHorizontalLabel', {
       defaultMessage: 'Horizontal bar',
     }),
   },
   {
     id: 'bar_stacked',
     icon: 'visBarVerticalStacked',
-    largeIcon: chartBarStackedSVG,
+    largeIcon: LensIconChartBarStacked,
     label: i18n.translate('xpack.lens.xyVisualization.stackedBarLabel', {
       defaultMessage: 'Stacked bar',
     }),
   },
+  {
+    id: 'bar_percentage_stacked',
+    icon: 'visBarVerticalStacked',
+    largeIcon: LensIconChartBarPercentage,
+    label: i18n.translate('xpack.lens.xyVisualization.stackedPercentageBarLabel', {
+      defaultMessage: 'Bar percentage',
+    }),
+  },
   {
     id: 'bar_horizontal_stacked',
     icon: 'visBarHorizontalStacked',
-    largeIcon: chartBarHorizontalStackedSVG,
+    largeIcon: LensIconChartBarHorizontalStacked,
     label: i18n.translate('xpack.lens.xyVisualization.stackedBarHorizontalLabel', {
       defaultMessage: 'Stacked horizontal bar',
     }),
   },
   {
-    id: 'line',
-    icon: 'visLine',
-    largeIcon: chartLineSVG,
-    label: i18n.translate('xpack.lens.xyVisualization.lineLabel', {
-      defaultMessage: 'Line',
+    id: 'bar_horizontal_percentage_stacked',
+    icon: 'visBarHorizontalStacked',
+    largeIcon: LensIconChartBarHorizontalPercentage,
```

Comment:
We can honestly also start getting rid of the distinction between `icon` and `largeIcon`. They should always display the custom one.

Reply:
Done, removed the `largeIcon` everywhere and used the custom icon for `icon` in all places.